src/services/etherspot.js

```diff
@@ -73,11 +81,25 @@ class EtherspotService {
       ? EnvNames.MainNets
       : EnvNames.TestNets;
 
-    this.sdk = new EtherspotSdk(privateKey, { env: envName, networkName });
-
-    await this.sdk.computeContractAccount({ sync: true }).catch((error) => {
-      reportErrorLog('EtherspotService init computeContractAccount failed', { error });
+    /**
+     * Cycle through the supported networks and build an
+     * array of instantiated instances
+     */
+    this.supportedNetworks.forEach((currentNetworkName) => {
+      // Instantiate
+      this.instances[currentNetworkName] = new EtherspotSdk(privateKey, { env: envName, currentNetworkName });
+
+      // Build up an array of computeContractAccount's to be executed later
+      etherspotComputeContractPromises.push(
+        this.instances[currentNetworkName].computeContractAccount({ sync: true }),
```

Comment:
I think we can just wait it here or wrap this as a whole promise, wdyt?

Reply:
Did you mean await on line 94? On line 101 we're executing all the promises in one go, if that's what you meant?